src/manager.js

```diff
@@ -8,7 +8,9 @@ var FORCED_STOP = 2;
  * @constructor
  */
 function Manager(element, options) {
-    options = options || {};
+
+    // make sure, options are copied over to a new object to prevent leaking it outside
+    options = extend({}, options || {});
```

Comment:
Why don't we use a ternary to avoid calling extend on an empty object. Also just for clarity since this change is designed leaking options use a new variable? Something like
```js
var newOptions = options ? extend({}, options) : {};
```